gcom_run.py

```diff
@@ -48,34 +47,34 @@ def gcom_object_archive_config():
             compiler=os.path.expanduser('~/.conda/envs/sci-fab/bin/gfortran'),
             common_flags=['-c', '-J', '$output']
         ),
-        ArchiveObjects(archiver='ar', output_fpath='$output/../libgcom.a'),
+        ArchiveObjects(archiver='ar', output_fpath='$output/libgcom.a'),
     ]
 
     return config
 
 
-def gcom_shared_object_config():
-    from fab.dep_tree import by_type
-
-    config = gcom_object_archive_config()
-    config.label = 'gcom shared object'
-
-    # don't pull the source again
-    config.grab_config = None
-
-    # compile with fPIC
-    fc: CompileFortran = list(by_type(config.steps, CompileFortran))[0]
-    fc.flags.common_flags.append('-fPIC')
-
-    cc: CompileC = list(by_type(config.steps, CompileC))[0]
-    cc.flags.common_flags.append('-fPIC')
-
-    # link the object archive
-    config.steps.append(LinkSharedObject(
-        linker=os.path.expanduser('~/.conda/envs/sci-fab/bin/mpifort'),
-        output_fpath='$output/libgcom.so'))
-
-    return config
+# def gcom_shared_object_config():
+#     from fab.dep_tree import by_type
+#
+#     config = gcom_object_archive_config()
+#     config.label = 'gcom shared object'
+#
+#     # don't pull the source again
+#     config.grab_config = None
+#
+#     # compile with fPIC
+#     fc: CompileFortran = list(by_type(config.steps, CompileFortran))[0]
+#     fc.flags.common_flags.append('-fPIC')
+#
+#     cc: CompileC = list(by_type(config.steps, CompileC))[0]
+#     cc.flags.common_flags.append('-fPIC')
+#
+#     # link the object archive
+#     config.steps.append(LinkSharedObject(
+#         linker=os.path.expanduser('~/.conda/envs/sci-fab/bin/mpifort'),
+#         output_fpath='$output/libgcom.so'))
+#
+#     return config
```

Comment:
Is there any value to this commented out code? If it serves no purpose please remove it.